apps/vh/bookings-api/demo.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: vh-bookings-api
+  namespace: vh
+spec:
+  releaseName: vh-bookings-api
+  values:
+    image: sdshmctspublic.azurecr.io/vh/bookings-api:prod-e23c232-20220117145616 #{"$imagepolicy": "flux-system:vh-bookings-api"}
+    environment: "demo"
+    resourceGroup: "vh-infra-core-demo"
+    subscriptionId: "c68a4bed-4c3d-4956-af51-4ae164c1957c"
+    tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+    managedIdentityClientId: "b526d7d6-d82c-45ed-8e42-b993c8c06f88"
```

Comment:
This should probably be the id for the corresponding demo managed identity - this looks to be dev